daemon/inertiad/crypto/symmetric.go

```diff
@@ -0,0 +1,63 @@
+package crypto
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+)
+
+const (
+	// SymmetricKeyLength is the length of the symmetric key in bytes
+	SymmetricKeyLength = 32
+)
+
+// getRandomNonce returns a random nonce for AES GCM mode
+func getRandomNonce(size int) []byte {
+	nonce := make([]byte, size)
+	rand.Read(nonce)
+	return nonce
+}
+
+// Encrypt encrypts plaintext using given key in AES GCM mode
+func Encrypt(key, plaintext []byte) ([]byte, error) {
+	block, err := aes.NewCipher(key)
+
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Minor: In Go you usually do
```go
something, err := expr()
if err != nil {
...
```

In other words, you generally want no whitespace between the call that returns the error and the error check so as to create clear groupings of error handling logic in your code.

Reply:
fixed